File: test/example/elixir/apps/publish/lib/publish.ex
defmodule Publish do
  def main([topic | _]) do
    HTTPoison.post!("http://localhost:8802/centro/v2/publish",
                    {:stream, IO.stream(:stdio, 100)},
                    [],
                    params: %{authz_token: System.get_env("CENTRO_TOKEN"),
                              topic: topic})
  end
end

File: test/example/python/publish.py
import requests, os, sys
params = {
    'authz_token': os.environ['CENTRO_TOKEN'],
    'topic': sys.argv[1]
}
requests.post('http://localhost:8802/centro/v2/publish',
              params=params,
              data=sys.stdin.buffer).raise_for_status()
